cli/cmd/lib_cluster_config.go

```diff
@@ -273,6 +273,11 @@ func setConfigFieldsFromCached(userClusterConfig *clusterconfig.Config, cachedCl
 	}
 	userClusterConfig.AvailabilityZones = cachedClusterConfig.AvailabilityZones
 
+	if userClusterConfig.SSLCertificateARN != nil && cachedClusterConfig.SSLCertificateARN != nil && *userClusterConfig.SSLCertificateARN != *cachedClusterConfig.SSLCertificateARN {
```

Comment:
Should this message also be shown if one of them is nil but the other one isn't?